File: JavaScript_12.18/1332 - Um-Dois-Três.js
/**
 * @author Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
 * @copyright Josélio Júnior (Lunatic Fox) 2022
*/

const l = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');
Array.from(Array(+l.shift()), _ => {
  const s = l.shift().replace(/\r/g, '');
  if (s.length === 3) {
    if (s.match(/^tw./) || s.match(/^t.o/) || s.match(/^.wo/))
      console.log(2);
    else console.log(1);
  } else console.log(3);
});

File: Lua_5.4/1332 - Um-Dois-Três.lua
--- author: Josélio de S. C. Júnior <joseliojrx25@gmail.com>
--- copyright: Josélio de S. C. Júnior 2022

for _ = 1, tonumber(io.read()) do
  local s = io.read()
  if #s == 3 then
    if s:match '^tw.' or s:match '^t.o' or s:match '^.wo' then
      print(2)
    else
      print(1)
    end
  else
    print(3)
  end
end